Dockerfile

```diff
@@ -8,7 +8,7 @@ LABEL Description="Tyk Dashboard docker image" Vendor="Tyk" Version=$TYKVERSION
 RUN apt-get update \
  && apt-get upgrade -y \
  && apt-get install -y --no-install-recommends \
-            curl ca-certificates apt-transport-https \
+            curl ca-certificates apt-transport-https gnupg npm \
```

Comment:
Sorry, why is this needed? Also `npm` should probably be installed further down after adding the custom nodejs repo. In addition to this worth uninstalling `npm` in the end as it's not really needed for the container to run I think.